src/gui/http_test.go

```diff
@@ -276,3 +277,54 @@ func TestGetBalanceHandler(t *testing.T) {
 		})
 	}
 }
+
+func TestEnableGUI(t *testing.T) {
+	tt := []struct {
+		name       string
+		enableGUI  bool
+		expectCode int
+	}{
+		{
+			name:       "disable gui",
+			enableGUI:  false,
+			expectCode: http.StatusNotFound,
+		},
+		{
+			name:       "enable gui",
+			enableGUI:  true,
+			expectCode: http.StatusOK,
+		},
+	}
+
+	for _, tc := range tt {
+		t.Run(tc.name, func(t *testing.T) {
+			c := Config{
+				EnableGUI:   tc.enableGUI,
+				DisableCSRF: true,
+				StaticDir:   "./static",
+			}
+
+			host := "127.0.0.1:6423"
```

Comment:
The mux method used in the other tests doesn't work here?

Reply:
Do you mean the `newServerMux` method?, it's created in the `Create` function.